python/ray/serve/_private/grpc_util.py

```diff
@@ -61,7 +63,7 @@ def create_serve_grpc_server(service_handler_factory):
         thread_pool=None,
         generic_handlers=(),
         interceptors=(),
-        options=(),
+        options=GRPC_OPTIONS,
```

Comment:
what are the options supported here? are we sure we want to share the configuration?

Reply:
There was a whole list of options that Ray currently supported https://github.com/ray-project/ray/blob/ca88ce7e6ac12644de56c1eb50491366711e0179/python/ray/util/client/common.py#L68 But good point let me make a Serve specific ones that's configurable as well. Will start with just the message size related ones.